polytracker/src/CMakeLists.txt

```diff
@@ -1,67 +1,61 @@
 set(CMAKE_CXX_STANDARD 17)
 
-add_subdirectory(passes)
-add_subdirectory(taintdag)
-
-# TODO Just do this here instead of in passes.
 find_package(LLVM 13 CONFIG)
 
-if (LLVM_FOUND)
+if(LLVM_FOUND)
   message(STATUS "LLVM_VERSION_MAJOR: ${LLVM_VERSION_MAJOR}")
   message(STATUS "LLVM_VERSION_MINOR: ${LLVM_VERSION_MINOR}")
   message(STATUS "LLVM_VERSION_PATCH: ${LLVM_VERSION_PATCH}")
 else()
-  # message(FATAL_ERROR "You need LLVM >=10 to build this (found LLVM ${LLVM_VERSION_MAJOR}).")
+  message(FATAL_ERROR "You need LLVM >=10 to build this (found LLVM
```

Comment:
It seems we're trying to find LLVM 13 (line 3) but the comment says >=10. Maybe we could get a version <13?

Reply:
> Maybe we could get a version <13?

Do you mean that we could lower the requirement to 10? Should probably just fix the comment to say 13.

Reply:
We need version 13 iirc. Fixing the comment will do it.